book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
```

Comment:
- [ ] To implement the has-many side of the relationship, a book has-many rentals, method for adding rentals should be present, something like so: 
```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```
Please kindly fix that